seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -708,9 +708,9 @@
       "id": "bad87fee1348bd9aedf08816",
       "title": "Link to External Pages with Anchor Elements",
       "description": [
-        "<code>a</code> elements, also known as <code>anchor</code> elements, are used to link to content outside of the current page.",
-        "Here's a diagram of an <code>a</code> element. In this case, the <code>a</code> element is used in the middle of a paragraph element, which means the link will appear in the middle of a sentence.",
+        "<code>a</code> elements, also known as <code>anchor</code> elements, are used to link to content inside, as well as outside of the current page. For this lesson, we are going to focus on using the <code>a</code> element for external links.",
```

Comment:
This is pedantic. `a` elements are used to link to content outside of the current page. Yes, they can also be used to link to content on the same page, but these are unnecessary details that will overload the learner.